Firebase/Database/Core/FRepoInfo.m

```diff
@@ -112,4 +112,22 @@ - (NSString *) connectionURLWithLastSessionID:(NSString*)lastSessionID {
     return url;
 }
 
+- (id)copyWithZone:(NSZone *)zone; {
+    return self; // Immutable
+}
+
+- (NSUInteger)hash {
+    NSUInteger result = host.hash;
+    result = 31 * result + (secure ? 1 : 0);
+    result = 31 * result + namespace.hash;
```

Comment:
Are we intentionally not including host in the hash?  This is technically fine (isEqual can be more precise than hash as long as they're not inconsistent) but I'm not sure there's any good reason for it.

Reply:
Yes, since the host can change " if a namespace gets assigned to a different backend server" (as per the original PR)

Reply:
Okay, I remembered some discussion around that...  but then we shouldn't use it in isEqual either, right?

Reply:
The Android client uses it in equal but not in hashCode. I can see that that makes sense in regards to the use in repoHashString.